pulp_python/app/tasks/sync.py

```diff
@@ -49,239 +49,160 @@ def sync(remote_pk, repository_pk, mirror):
         raise serializers.ValidationError(
             detail=_("A remote must have a url attribute to sync."))
 
-    first_stage = PythonFirstStage(remote)
-    DeclarativeVersion(first_stage, repository, mirror).create()
-
-
-class PythonFirstStage(Stage):
-    """
-    First stage of the Asyncio Stage Pipeline.
-
-    Create a :class:`~pulpcore.plugin.stages.DeclarativeContent` object for each content unit
-    that should exist in the new :class:`~pulpcore.plugin.models.RepositoryVersion`.
-    """
-
-    def __init__(self, remote):
-        """
-        The first stage of a pulp_python sync pipeline.
-
-        Args:
-            remote (PythonRemote): The remote data to be used when syncing
-
-        """
+    with NamedTemporaryFile() as config:
+        config.write(formatted_config(remote))
+        config.seek(0)
+        first_stage = PythonBanderStage(remote, config)
+        DeclarativeVersion(first_stage, repository, mirror).create()
+
+
+def formatted_config(remote):
```

Comment:
We should probably be trying to set these values directly on the configuration object rather than writing a file that then gets parsed.  I'm not sure if that is currently supported but it's something we should probably work towards.